test/mocha/helpers.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {agent} from '@bedrock/https-agent';
+import {httpClient} from '@digitalbazaar/http-client';
+
+const headers = {
+  Accept: 'application/ld+json;profile="https://w3id.org/did-resolution"'
+};
+
+export const makeRequest = async ({did}) => {
+  const basePath = '/1.0/resolve/identifiers/';
```

Comment:
Is this required by the DID Resolution spec, feels off to me. `/1.0/identifiers/resolve` feels a bit better.

Reply:
no and not not sure where that came from. The spec gives this example: https://dev.uniresolver.io/1.0/identifiers/



So we can make probably just do that.

Reply:
Changed the route here: https://github.com/digitalbazaar/bedrock-did-resolver-http/pull/3/commits/3364e4a7a044e47eebb31db52e4a69a83d1d19d3